papermario/src/code_111f0_len_860.c

```diff
@@ -1,6 +1,10 @@
 #include "common.h"
 
-INCLUDE_ASM(code_111f0_len_860, func_80035DF0);
+
+void func_80035DF0(s16 arg0) {
+    s16* tempPtr = &D_800A0942;
+    D_800A0942 = arg0;
```

Comment:
could you use tempPtr here? I know it's not necessary, but since we're defining it we should use it.